codecov.yml

```diff
@@ -0,0 +1,8 @@
+---
+coverage:
+  notify:
+    slack:
+      default:
+        url: secret:5WK4lHFGNbrytMXL36g63my9P4W6kLCI3sP/vpKELSubtWw0aU4u4wRAa1yb6co5zFhJicC3t4dxZvPhQ63ZXIVVUmqkykam4wjfMguN+GDo7IxUMygmyWzxdzCwLZBRomOS1F2UCvDS29zBfNiF7LudEyUMPuKtpo5lj4jv5tU=
+
+comment: false
```

Comment:
Don't place a comment on the PR. I did this because I expected it to show up as a check below, but that didn't happen. I can turn this back on if desired.